### DISCUSSION TITLE
Sharing _variables.yml accross multiple books: is it possible?

### DISCUSSION THREAD

**Discussion by cavo789** [quarto-dev/quarto-cli#11548] [Q&A]
### Description

Hi all

I'm a happy user of Quarto and I use it both on my local dev machine (to generate an HTML site) and on my CI/CD workflow (to generate docx and PDFs).

By pushing my codebase to Gitlab, I detect if changes have been done in my /documentation folder and if so, render books. Really nice, really wow !

but ... right now, I should put the _variables.yml file in each sub-folder and a lot of variables are in common.

```text
Folder /documentation
├── developer_guide
│   ├── _quarto.yml
│   ├── _variables.yml
│   ├── index.md
├── installation_guide
│   ├── _quarto.yml
│   ├── _variables.yml
│   ├── index.md
├── operational_guide
│   ├── _quarto.yml
│   ├── _variables.yml
│   ├── index.md
```

Is there any "way" to put the file in my documentation folder like illustrated below and tells to the _quarto.yml file to retrieve it one folder up?

```text
Folder /documentation
├── _variables.yml
├── developer_guide
│   ├── _quarto.yml
│   ├── index.md
├── installation_guide
│   ├── _quarto.yml
│   ├── index.md
├── operational_guide
│   ├── _quarto.yml
│   ├── index.md
```

(On my local machine creating symlinks will certainly solve this issue but symlink will not be available on the Gitlab server when running the CI)

Many thanks!

**Comment by mcanouil** [comment]
No, it's not.
One of the main design choices in Quarto is that a Quarto project HAVE TO be self-contained for reproducibility reasons, *i.e.*, Quarto resolves (mostly) all paths starting with `/` as "root of the project" (not as "root of the file system").

**Comment by cavo789** [comment]
Thanks!  Yeah, it was well my fear and your decision is legitimate.  I'll then continue to duplicate that file in both folders.

I really enjoy to be able to write my documentation using Quarto and to render it in different formats 💯
